//
// Created by Earl Leonarda on 15/02/2020.
//

#include "task3.hpp"

void	case_of_error(const string& str)
{
	cout << str << endl;
	exit(1);
}

